video/randominput.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+// BoB robotics includes
+#include "input.h"
+
+// OpenCV
+#include <opencv2/opencv.hpp>
+
+// Standard C++ includes
+#include <random>
+
+namespace BoBRobotics {
+namespace Video {
+//! A Video::Input which generates white noise, for testing purposes
+class RandomInput : public Input {
```

Comment:
As it's header-only anyway, you **could** template on the type of generator.

Reply:
That's true. I did think about this, but the only thing is that I can't seem to find any other generators that would work here (i.e. return ``uchar``s in the right range). I guess you could provide a custom generator though?

Reply:
``std::uniform_int_distribution<uchar>`` is totally the only distribution that makes sense but different RNGs could plausible be useful (basically cos MT is slow)